CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 - Fix: changed the default value of `-dbTimeout` to 0 to resolve conflict with `-dbURI` (#4496) 
 - Fix: wrong INFO startup log showing ORION_MONGO_TIMEOUT, ORION_IN_REQ_PAYLOAD_MAX_SIZE and ORION_OUT_REQ_MSG_MAX_SIZE env var values (#4496)
 - Fix: return 400 Bad Request when subject.entities exists but it is an empty array (#4499)
+- Fix: Attribute deleting script is not working (#4508)
```

Comment:
```suggestion
```

Utility scripts modifications are not tracked in CHANGES_NEXT_RELEASE file.

Reply:
Fixed in 498d133